### DISCUSSION TITLE
page not found for next swc related packages

### DISCUSSION THREAD

**Discussion by yoonjaepark** [vercel/next.js#54487] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:28 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6000
    Binaries:
      Node: 18.6.0
      npm: 8.18.0
      Yarn: 3.6.2
      pnpm: N/A
    Relevant Packages:
      next: 13.4.19
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
      typescript: N/A
    Next.js Config:
      output: N/A
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Package manager (npm, pnpm, Yarn)

### Link to the code that reproduces this issue or a replay of the bug

yarn && next build

### To Reproduce

yarn beery monorepo
docker yarn pacage app build error log

2.147 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
2.147 https://nextjs.org/telemetry
2.147 
2.216 - info Creating an optimized production build...
2.274 - warn Attempted to load @next/swc-linux-arm64-gnu, but it was not installed
2.274 - warn Attempted to load @next/swc-linux-arm64-musl, but it was not installed
3.080 - error Failed to load SWC binary for linux/arm64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc

### Describe the Bug

https://yarnpkg.com/package/@next/swc-linux-arm64-musl
https://yarnpkg.com/package/@swc/core-linux-arm64-gnu
page not found.
need to check the package.

![image](https://github.com/vercel/next.js/assets/6939460/0ad9edac-b321-4d55-91f0-81aa9e439388)

### Expected Behavior

build error

### How are you deploying your application? (if relevant)

docker

**Comment by balazsorban44** [comment]
The whole site seems to be broken: https://yarnpkg.com/package/

![image](https://github.com/vercel/next.js/assets/18369201/7e0172cf-d46e-4fca-ab6a-acb9e074179a)


The source code for the package can be found here: https://github.com/vercel/next.js/tree/canary/packages/next-swc

You can see the specific packages here:

https://www.npmjs.com/package/@next/swc-linux-arm64-musl
https://www.npmjs.com/package/@next/swc-linux-arm64-gnu

If you have any issues, please report a bug with a reproduction.
